web-bundle/src/main/java/com/graphhopper/replica/ReplicaCustomSpeedsFootTagParser.java

```diff
@@ -0,0 +1,62 @@
+package com.graphhopper.replica;
+
+import com.google.common.collect.ImmutableMap;
+import com.graphhopper.customspeeds.CustomSpeedsUtils;
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.ev.EdgeIntAccess;
+import com.graphhopper.routing.ev.EncodedValueLookup;
+import com.graphhopper.routing.util.parsers.FootAverageSpeedParser;
+import com.graphhopper.util.PMap;
+
+public class ReplicaCustomSpeedsFootTagParser extends FootAverageSpeedParser {
```

Comment:
Unfortunately, this class proved messier to override than the bike/car-specific classes were, because some key speed-handling functions there aren't private/protected. The minimal approach I could find involved overriding `setSpeed()`, which required copy/pasting a few other related methods. I left comments where I directly copied from the parent class

Reply:
ah, that's unfortunate. i'm on board proceeding with some copy/pasting here, but i don't love it. as i'm sure you're aware, every additional copy/paste makes core updates more expensive since we need to manually update those sections (and ensure we find or remember all the relevant spots!). we've discussed this before, but ideally we could contribute to core graphhopper and make the necessary refactors to allow for cleaner overrides in our fork. 

i don't think we should block on that here, and it could be a bit more challenging given we're still on a branch of core graphhopper...but if we never take the time to clean this up, it'll just get harder and harder to pull in core updates, and core updates are already pretty time consuming. maybe we can advocate for some time to improve this (and similar spots from https://github.com/replicahq/graphhopper/pull/143#pullrequestreview-1440076896 and https://github.com/replicahq/graphhopper/pull/112) once the scenario work calms down?